catalog.html

```diff
@@ -152,8 +153,8 @@ <h3 class="hotel-card-title">Amara Resort & Spa</h3>
             </a>
             <b class="hotel-card-price">от 4000 ₽ </b>
             <a href="#" class="hotel-card-button">Подробнее</a>
-            <a href="#" class="hotel-card-button button">В избранное</a>
-            <img class="hotel-rating-star" src="" alt="">
+            <a href="#" class="hotel-card-button-blue ">В избранное</a>
```

Comment:
Можно намного проще. У нас есть класс `.button`. Используй его для начала. А затем второй для цвета:
```suggestion
            <a href="#" class="button button-blue ">В избранное</a>
```

Такая же логика для кнопки "Подробнее". Только ей второй класс не нужен, у нее цвет по умолчанию

Reply:
Добавила, но все равно не понимаю зачем нужен второй по сути дублирующий класс.